packages/fether-react/src/stores/createAccountStore.js

```diff
@@ -55,11 +56,13 @@ export class CreateAccountStore {
    */
   @action
   clear = async () => {
```

Comment:
not related to this PR: found a theoretical bug afaik

currently `clear` is async but called simply as `this.clear();` in the other functions of the class, before they changed the properties to their new values. I would have assumed that `this.clear()` would be pushed on the event loop async stack and be run after the end of the calling function (and provoke a bug), however this is not the case; need to research why..

i.e. completely confused by the output of this:
```js
let a = 3;

let clear = async () => {
 console.log('inside clear');
 a = 0;
}

let run = async () => {
 console.log('inside run');
 clear();
 console.log('inside run - continuing');
 a = 4;
}

run();
console.log('post run');

// output:
// inside run
// inside clear
// inside run - continuing
// post run

// would have expected, because of the way sync/async calls and promises are handled in js:
// post run
// inside run
// inside run - continuing
// inside clear
```

need to brush up on my js

all the same let's remove `async` from clear and make it a regular function!

Reply:
ok, this explains that: `The callback passed to a Promise constructor is always called synchronously, but the callbacks passed into then are always called asynchronously`